client.go

```diff
@@ -131,6 +131,15 @@ func (c *Client) Query(cql string, values ...any) *Query {
 	}
 }
 
+// Batch creates a new Astra batch query.
+func (c *Client) Batch(batchType BatchType, queries ...*Query) *BatchQuery {
```

Comment:
I think it makes more sense to not include `BatchType` in the parameters and default to logged. Internally, this should default to `Logged` and is most likely correct choice for most cases, multi-partition mutations will be logged and single partitions will not be logged. 

Then, `BatchQuery` should provide a method `.BatchType(batchType BatchType)` like you did for `.Keyspace()`.

```go
func (b *BatchQuery) BatchType(batchType BatchType) *BatchQuery {
	b.batchType = batchType
	return b
}
```

Reply:
I completely agree with defaulting to "LOGGED" unless specified differently.  And the use cases for "UNLOGGED" are (should be) small in number and infrequent.